decidim-generators/lib/decidim/generators/app_generator.rb

```diff
@@ -158,11 +162,24 @@ def remove_default_error_pages
       def decidim_initializer
         copy_file "initializer.rb", "config/initializers/decidim.rb"
 
+        gsub_file "config/environments/production.rb",
+                  /config.log_level = :debug/,
+                  "config.log_level = %w(debug info warn error fatal).include?(ENV['RAILS_LOG_LEVEL']) ? ENV['RAILS_LOG_LEVEL'] : :debug"
+
+        gsub_file "config/environments/production.rb",
+                  %r{# config.action_controller.asset_host = 'http://assets.example.com'},
+                  "config.action_controller.asset_host = ENV['RAILS_ASSET_HOST'] if ENV['RAILS_ASSET_HOST'].present?"
+
         if options[:force_ssl] == "false"
           gsub_file "config/initializers/decidim.rb",
                     /# config.force_ssl = true/,
                     "config.force_ssl = false"
         end
+        if options[:locales].present?
+          gsub_file "config/initializers/decidim.rb",
```

Comment:
After introducing this, when generating the development_app I have this:

```ruby
  #
  # When an organization is created through the System area, system admins will
  # be able to choose the available languages for that organization. That list
  # of languages will be equal or a subset of the list in this file.
  config.available_locales = Rails.application.secrets.decidim[:available_locales].presence || [:en]
  # Or block set it up manually and prevent ENV manipulation:
  config.available_locales = %w(en ca es)
```

It looks a bit weird. Should it look like this, maybe? 

```ruby
  #
  # When an organization is created through the System area, system admins will
  # be able to choose the available languages for that organization. That list
  # of languages will be equal or a subset of the list in this file.
  # config.available_locales = Rails.application.secrets.decidim[:available_locales].presence || [:en]
  # Or block set it up manually and prevent ENV manipulation:
  config.available_locales = %w(en ca es)
```

(I'm not 100% sure)

Reply:
Good catch! It should be commented now.
BTW, This needs to be done for the test app as many tests depends on having exactly those 3 languages defined.
For the development_app is just for the sake of not having a cumbersome list of them.